simpl/config.py

```diff
@@ -526,7 +525,7 @@ def get_defaults(self):
         parsed, _ = parser.parse_known_args([])
         return vars(parsed)
 
-    def parse_ini(self, paths=None, namespace=None):
+    def parse_ini(self, paths=None, namespace=None, permissive=False):
```

Comment:
I added the permissive argument, but there's not really any easy way to use it, unless you were to subclass Config and override the `load_options` method. 

I assumed (and agree) that throwing an error upon encountering "loose" options might not be the desired behavior for some. For the record, this works very well for our use cases, and would have alerted us to problems sooner, although I would argue that it would be nice if you could turn this feature on more easily, i.e. the `permissive` feature of our ini parsing.